packages/avatar-bundle/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@hig/avatar-bundle",
+  "version": "1.0.0",
+  "description": "HIG Avatar Bundle",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/index.js",
+  "module": "build/index.es.js",
+  "files": [
+    "build/*"
+  ],
+  "dependencies": {
+    "@hig/utils": "^0.4.0",
+    "prop-types": "^15.7.1",
+    "react-lifecycles-compat": "^3.0.4"
+  },
+  "peerDependencies": {
+    "@hig/avatar": "^1.3.0",
```

Comment:
wondering if this makes more sense to be in the `dependencies` section?

Reply:
It does, not sure why it's there instead. I'll update it.